bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/TargetEnvironment.java

```diff
@@ -232,17 +234,22 @@ public boolean containsTarget(SPTarget target) {
     }
 
     /**
-     * Gets a list of all the guide groups in this environment
+     * Get a list of all the SPCoordinates objects in this environment, in the
+     * order:
```

Comment:
Looking over the uses of this and of `Asterism.allSPCoordinatesJava()`, I can see that it doesn't matter. I think I thought at some point that it would matter, but it turned out that it was unimportant. I'll fix the comment to reflect this.